be/src/exec/vectorized/hdfs_scanner_text.h

```diff
@@ -20,24 +20,7 @@ class HdfsTextScanner final : public HdfsScanner {
     Status _parse_csv(ChunkPtr* chunk);
 
 private:
-    class HdfsScannerCSVReader : public CSVReader {
```

Comment:
These change may also need to be described in the commit message

Reply:
Oh. This is a simple refactor to reduce exposed class in header file. This class is only be used in cpp file.